src/main/java/com/minecolonies/colony/Schematics.java

```diff
@@ -219,7 +291,12 @@ public static int getMaxLevelForHut(String hut)
     @SideOnly(Side.CLIENT)
     public static void setStyles(Map<String, List<String>> hutStyleMap, Map<String, List<String>> decorationStyleMap)
     {
-        Schematics.hutStyleMap = hutStyleMap;
-        Schematics.decorationStyleMap = decorationStyleMap;
+        /*todo this isnt right.  we need to look at the build tool
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)